lib/src/data/models/movie_page_model.dart

```diff
@@ -0,0 +1,21 @@
+import 'movie_model.dart';
+
+class MoviePageModel {
+  late final int page;
+  late final List<MovieModel> results;
+  late final int totalPages;
+  late final int totalResults;
```

Comment:
why is defined as late?

Reply:
That was because i didn't make a factory, so it was declared as a late attribute, i changed that to make a factory method and return a MoviePageModel